install_deps.sh

```diff
@@ -91,19 +117,19 @@ then
 		. ./scripts/install_openxr.sh
```

Comment:
You've only utilized `illixr_nproc` in some of the builds. Is this because you don't expect `OpenXR-SDK`. `gtest`, ... to run out of memory?

Reply:
forgot about those, lol